### ISSUE TITLE
Our Windows CI is broken

### ISSUE THREAD

**Issue by alat-rights** [deepchem/deepchem#2672]
## 🐛 Bug

In test for DeepChem core (and other tests), the Windows CI fails with `Error: The process 'C:\Miniconda3\condabin\conda.bat' failed with exit code 1` in the dependency installation phase.

## To Reproduce

1. Push any change to a DeepChem PR to trigger a GH Actions run

https://github.com/deepchem/deepchem/pull/2642/checks?check_run_id=3385813345

## Expected behavior

The error shouldn't occur.

## Environment

* GitHub Actions CI in the DeepChem GitHub repo

**Comment by rbharath** [comment]
Thanks for the report! This looks serious. Does anyone have any ideas for why this could be happening?

**Comment by alat-rights** [comment]
It seems like the cause was a pip failure

```
Downloading opt_einsum-3.3.0-py3-none-any.whl (65 kB)
  
  
  failed
  Warning: 
  CondaEnvException: Pip failed
```

**Comment by rbharath** [comment]
Good find! Hmm, not sure what that's a dependency of. TF/Torch/Jax?

**Comment by arunppsg** [comment]
It is due to a jax dependency.
Installation command:
```
conda-merge env.common.yml env.cpu.yml env.test.yml > env.yml
conda env create -f env.yml
``` 
This is the error message from installation:
```
Collecting opt_einsum
Using cached opt_einsum-3.3.0-py3-none-any.whl (65 kB) 
Pip subprocess error:    
ERROR: Could not find a version that satisfies the requirement jaxlib==0.1.68 (from jax[cpu]->-r C:\Users\DELL\Documents\condaenv.2k_malha.requirements.txt (line 10)) (from versions: none)  
ERROR: No matching distribution found for jaxlib==0.1.68 (from jax[cpu]->-r C:\Users\DELL\Documents\condaenv.2k_malha.requirements.txt (line 10)) 

failed
CondaEnvException: Pip failed                                                    
```

#2683 might also be linked to this.

**Comment by rbharath** [comment]
@VIGNESHinZONE Would you have any ideas what might be happening here with the Jax install?

**Comment by VIGNESHinZONE** [comment]
`Deepchem core` workflow can always be ignored as its main usage is to check if doctests are passing. 

`Jax` does not currently support windows build, hence the results are expected.

There are currently 5 main GitHub workflows - 

1. `deepchem common` - which checks only functionalities like `Dataset`, `Featurisers`, `Molnet`, etc which are independent of any Neural Network libraries.
2. `deepchem torch` - it is used to check the Deepchem torch models and their pytests. (We do not check the doctest here)
3. `deepchem tensorflow` - it is used to check the Deepchem tensorflow models and their pytests. (We do not check the doctest here again)
4. `deepchem jax` - it is used to check the Deepchem jax models and their pytests. (We do not check the doctest here again)

5. `deepchem core`- Here we install all dependencies of Pytorch, tensorflow, jax into a single environment. We run the doctests here because its hard to doctests using pytests with markers mentioned [here](https://github.com/deepchem/deepchem/blob/085a3e53fb69b3108de0527afa1f144e1b861108/setup.cfg#L2)


The first four workflows are the only important ones. Let's remove windows tests from `deepchem core` checks.

**Comment by RMeli** [comment]
> Jax does not currently support windows build, hence the results are expected.

I think this error might not be only related to Windows. I get the same error when trying to install DeepChem ([from source with conda](https://deepchem.readthedocs.io/en/latest/get_started/installation.html#from-source-with-conda)) on Linux (#2683).

**Comment by VIGNESHinZONE** [comment]
Yes my bad, even Im able to reproduce even with the lightweight install. Im unsure what's causing this error, maybe I will start an issue with the `Jax`.

**Comment by rbharath** [comment]
Windows is still unstable but closing for now since I think no updates
